shell/lib/asdf.sh

```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env bash
 # Utilities for working with asdf
 
+# asdf_global_plugins is a list of global plugins to always make sure exist, usually
+# to ensure that /other/ asdf plugins have a valid build environment to build
+# themselves with.  This script will attempt to _first_ install everything from this
+# global plugins list before going through the tool-versions from the local app.
+# Any global plugin that has a matching version from inside the app will set that
+# version globally.  If the app does not have the global plugin in its local
+# tool-versions file, it will fall back to the hardcoded version in here.
+asdf_global_plugins=("golang 1.19.5")
```

Comment:
Can the trade-off for this "global" version be that we read it from the `devbase` `.tool-versions` file? That's what we've been using, more or less, for global versions and would be easily updatable then.